qemu/tests/cfg/virtio_driver_sign_check.cfg

```diff
@@ -1,16 +1,44 @@
 - virtio_driver_sign_check:
     virt_test_type = qemu
-    only Win7.x86_64
-    # This script only need run once for every virtio driver version.
-    # So need not add to our testing loop.
+    only Windows
     type = virtio_driver_sign_check
-    cdroms += " sdk virtio"
-    cdrom_virtio = isos/windows/virtio-win.iso
-    cdrom_sdk = isos/windows/GRMSDKX_EN_DVD.iso
-    floppies = "fl"
-    floppy_name = images/win7-64/virtio-win.vfd
-    list_files_cmd = dir /s /b %s | find "%s"
-    winsdk_au3 = autoit/winsdk.au3
-    signtool_install = "D:\autoit3.exe c:\winsdk.au3"
-    signtool_cmd = "c:\Program Files\Microsoft SDKs\Windows\v7.0\Bin\signtool.exe" verify /v /kp /c %s %s
-    drive_list = F: A:
+    list_files_cmd = 'dir /s /b %s | find /i "%s" | find "%s"'
+    Win2008..sp2:
+        # windows 2008 not support virtio input device
+        no with_vioinput
+        signtool_path_key = "WLK"
+    Win2008..r2, Win7, Win8, Win2012:
+        signtool_path_key = "HCK"
+    Win10, Win2016:
+        signtool_path_key = "HLK"
```

Comment:
Suggest to make `signtool_path_key = "HLK"` be default:
```
signtool_path_key = "HLK"
Win2008..sp2:
    no with_vioinput
    signtool_path_key = "WLK"
Win2008..r2, Win7, Win8, Win2012:
    signtool_path_key = "HCK"
```

Reply:
thanks for you advise, has updated~